content/testnet.md

```diff
@@ -0,0 +1,152 @@
+### 3. Clone the RubixGoPlatform Repository
+
+- Clone the repository:
+    ```bash
+    git clone https://github.com/rubixchain/rubixgoplatform.git
+    ```
+
+- Navigate to the repository directory
+    ```bash
+    cd rubixgoplatform
+    ```
+
+### 4. Build the RubixGoPlatform Executable
+- For Linux : 
+    ```bash
+    make compile-linux
+    ```
+
+- For Windows : 
+    ```bash
+    make compile-windows
+    ```
+
+- For MAC : 
+    ```bash
+    make compile-mac
+    ```
+
+Upon successful compilation, the rubixgoplatform executable will be created in a folder corresponding to your operating system. For example, on Linux, it will be located in the ./linux directory.
+
+### 5. Prepare the Necessary Files
+
+Ensure the following files are in the build directory:
+- rubixgoplatform executable
+- ipfs executable (from Kubo v19)
+- testswarm.key (provided by the Rubix team)
+
+
+## Joining the Testnet
+
+### Start a Rubix Node
+
+1. Open a terminal and navigate to the directory containing the `rubixgoplatform` executable.
+
+   ```bash
+   cd /path/to/rubixgoplatform
+   ```
+
+2. Start a Rubix node with the following command:
+   ```bash
+   ./rubixgoplatform run -s -testNet -p <node_name> -n <node_number> -grpcPort <grpc_port>
+    ```
+    
+    Replace the placeholders:
+
+    - \<node\_name\>: A unique name for your node.
+
+    - \<node\_number\>: Numeric identifier for the node.
+
+    - \<grpc\_port\>: Port number for gRPC communication.
+
+### Confirm the node is operational and connected to the testnet. Check for below in the logs.
+
+![Bootstrap connection check](/images/bootstrap_verify.png)
+
+
+### Create a DID (Decentralized Identifier) for the Node and register the DID
+
+1. Once the node is running, create a DID by executing:
+
+   ```bash
+   ./rubixgoplatform createdid -didType 4 -port <port_number>
+   ```
+
+2. Register the DID created by executing:
+   ```bash
+   ./rubixgoplatform registerdid -did <did_created> -port <port_number>
+    ```
+
+### Generate Test RBT Tokens
+To get test RBTs (Rubix Blockchain Test Tokens):
+ - Open the [faucet webpage](www.faucetlink.com) in browser
+ - In the address field, provide your DID created above to receive test tokens for your node, and submit DID.
+
+
+## Verification
+Validate token generation by checking the node’s token balance. 
+To check balance of a particular DID:
+
+   ```bash
+   ./rubixgoplatform getaccountinfo -did <did> -port <port_number>
+   ```
+Congratulations! You have successfully joined the Rubix testnet. You can now use the node for testing and development purposes.
+
+## Next Steps
+
+Now that you have successfully joined the Rubix testnet, you can:
+- Explore Rubix’s testnet functionality, such as token transfers, smart contracts etc.
+- Visit the [Rubix Documentation Portal](https://learn.rubix.net/) to learn more or contact the Rubix team for further support.
+- Use the [Rubix Block Explorer](www.explorerlink.com) to monitor transactions and node activity
```

Comment:
Provide explorer link

Reply:
Again, explorer link is not ready. This is a placeholder.